src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -15,35 +15,89 @@
  * Class that allows you to exercise Heng Li's fermi-lite assembler.
  * This is backed by JNI native code, which could fail to load.
  */
-public final class FermiLiteAssembler {
+public final class FermiLiteAssembler implements AutoCloseable {
     private static volatile boolean nativeLibLoaded = false;
+    private final ByteBuffer opts;
 
     public FermiLiteAssembler() {
         loadNativeLibrary();
+        opts = createDefaultOptions();
+        opts.order(ByteOrder.nativeOrder()).position(0).limit(opts.capacity());
     }
 
+    @Override
+    public void close() { destroyByteBuffer(opts); }
+
     public interface BasesAndQuals {
         byte[] getBases();
         byte[] getQuals();
     }
 
+    // number of threads; don't use multi-threading for small data sets
```

Comment:
these comments should be method javadoc so clients can are more likely to see them.